docs/Users_Guide/reformat_stat_data.rst

```diff
@@ -7,33 +7,40 @@ Description
 ===========
 
 The METreformat module provides support for reformatting/rearranging MET .stat/.tcst output into METplotpy-readable
-input formats.  The MET .stat/.tcst ASCII output generated by MET tools such as the point-stat, grid-stat,
-ensemble-stat, and tc-pairs tools may contain ASCII columnar data from multiple line types (determined by settings in the
-MET configuration file).
+input formats.  The formats vary, based on the plot type (e.g. a histogram plot requires ranks and values column data,
+whereas a line plot will need statistic name and statistic value columns). The MET .stat/.tcst ASCII output generated
+by MET tools such as the point-stat, grid-stat, ensemble-stat, and tc-pairs tools may contain ASCII columnar data
+from multiple line types (determined by settings in the MET configuration file).
 
-Currently, there are MET linetypes that **do not** have reformatter support.  When an unsupported linetype
-is requested, an appropriate error message will be generated.
+.. note ::
+  Currently, there are MET linetypes that **do not** have reformatter support.  When an unsupported linetype
+  is requested, an appropriate error message will be generated.
 
-In the MET .stat and .tcst files, there are numerous rows of data that have a different number of columns.
+MET .stat and .tcst files can be comprised of numerous rows of data with different number of columns. This is a
+result of requesting multiple linetypes in a MET configuration file (e.g. requesting .
```

Comment:
@bikegeek This line and the next line currently read:
"This is a result of requesting multiple linetypes in a MET configuration file (e.g. requesting . All MET .stat line types have common columns".  Is that how it should read?  I'm confused by the ". All" part and am wondering if it should be "This is a result of requesting multiple linetypes in a MET configuration file (e.g. requesting all MET .stat line types have common columns" instead?